### ISSUE TITLE
 Dotted line shows at the bottom after convert. 

### ISSUE THREAD

**Issue by cyfyifanchen** [wulkano/kap#1200]
**macOS version:** 14.1.1 (23B81)
**Kap version:** Version 3.6.0 (3.6.0.1846)

#### Steps to reproduce
1. Start Kap with select area.
2. Start recording.
3. Convert the recording.

#### Current behaviour
1. It shows a dotted line at the bottom of the gif(see below gif) sometimes, I tried a few times, only one time it worked without the line.

![bug-sample](https://github.com/wulkano/Kap/assets/2340896/7d0ee7b4-b8e8-4435-8e8b-277857c8aaec)

#### Expected behaviour
1. It is supposed to work out without the line. 

#### Workaround
It does seem like the problem can be gone if keep trying.

**Comment by design-ninja** [comment]
Same

**Comment by cyfyifanchen** [comment]
Right, it's inconsistent. But I do feel like it happens more when selection is smaller(e.g less than 1000 * 1000)

**Comment by rymiles** [comment]
Same issue here
